app/Http/Controllers/Backend/BranchController.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use App\Http\Controllers\Controller;
+use App\Models\Branch;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Session;
+
+class BranchController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+
+        $branchd=Branch::where('deleted_at',Null)->get();
```

Comment:
remove where condition and use softdelete libraray

Reply:
All done sir